docs/linting/README.md

```diff
@@ -68,14 +77,43 @@ dist
 
 ## Running ESLint
 
-With that configured, open a terminal to the root of your project, and run the following command
+With that configured, open a terminal to the root of your project, and run the following command:
 
-```bash npm2yarn
-npm run eslint . --ext .js,.jsx,.ts,.tsx
+<Tabs groupId="npm2yarn">
+<TabItem value="npm">
+
+```bash
+npx eslint . --ext .js,.jsx,.ts,.tsx
+```
+
+</TabItem>
+<TabItem value="Yarn">
+
+```bash
+yarn eslint . --ext .js,.jsx,.ts,.tsx
 ```
 
+</TabItem>
+</Tabs>
+
 That's it - ESLint will lint all `.js`, `.jsx`, `.ts`, and `.tsx` files within the current folder, and will output the results to your terminal.
 
+You are also recommended to add an npm script in your package.json, so you don't have to repeat the same command every time you run ESLint.
+
+```json title="package.json"
+{
+  "scripts": {
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  }
+}
+```
+
+This way, you can invoke the `lint` script directly:
+
+```bash npm2yarn
```

Comment:
Random thought: it'd be nice to have pnpm support. Doesn't need to be in this PR though, I can file a followup issue for later if this PR is merged without it.

Reply:
It will require changing all the other `npm2yarn` tabs, though. You can even request that in Docusaurus.

Reply:
@JoshuaKGoldberg that's actually great idea